static/styles/compose.css

```diff
@@ -468,6 +468,10 @@ input.recipient_box {
     margin-top: 10px;
 }
 
+.formatting_button {
+    font-size: 12px !important;
```

Comment:
Why does this need `!important`?  I would hope we could just have a more strongly scoped selector, like `compose_control_button.formatting_button` or something.

Also, we've been trying to use e.g. `font-size: 80%` for scaling icons, since that's going to work better when we change the app's font size.

Reply:
Not relevant anymore since we moved to fa icons for formatting buttons.